compiler/damlc/tests/src/DA/Test/DataDependencies.hs

```diff
@@ -2172,19 +2204,19 @@ tests tools = testGroup "Data Dependencies" $
           dar proj = path proj </> proj <.> "dar"
           step proj = step' ("building '" <> proj <> "' project")
 
-          damlYamlBody name dataDeps = unlines
+          damlYamlBody name useScript dataDeps = unlines
```

Comment:
minor suggestion: I think the callsites would read better if this were

```suggestion
          damlYamlBody name extraDeps dataDeps = unlines
```

so tests that don't use `daml-script` call it with an empty list while tests that do pass a singleton list `[scriptDar1dev]`